doc/whats_new.rst

```diff
@@ -15,13 +15,18 @@ Changelog
 BUG
 ~~~
 
-    -
+    - :func:`compute_raw_psd` and :func:`compute_epochs_psd` no longer subset the SSP matrix with picks before applying SSP projectors by `Chris Holdgraf`_.
+
+    - :func:`mne.Epochs.plot_psd` no longer calls a Welch PSD, and instead uses a Multitaper method for the PSD. Flags for this function have been updated accordingly to be passed to :func:`mne.time_frequency.psd_multitaper` by `Chris Holdgraf`_
```

Comment:
it's a bit cryptic at the moment why these two were bugs. Can you rephrase it so that it's more clear?